### PR TITLE
Navbar & Route updates

### LINKED ISSUE #410: Clean up routes in web.php
## Expected Behavior
If i visit a page that does not have a purpose (such as /bookings/rooms), it should show a 404. 
## Current Behavior
500 error showing that this pages does not have a controller attached but there is a route leading to it. 
## Possible Solution

only have the right routes listed in web.php

# Bug Submission Checklist
- [ ] Tagged commits fixing the bug for traceability
- [ ] Tagged commits updating tests to cover the reported bug

### LINKED ISSUE #415: Refactor & Update Navbar
### What needs to be done
Navbar needs to be update to display links to the new pages as well as only show the pages the current user has access too.

### Out of scope
Anything not explicitly required to achieve the above

### PR DESCRIPTION
**Pull request by alexstojda**
##### _List all issues this PR resolves_
- Closes #415
- Closes #410 

##### _Complete these items before requesting reviews_
- [x] Static Analysis Rules Validated
- [x] All commits follow naming conventions

### Sub-tasks Checklist
- [x] UI Test
- [x] Backend Tests
- [x] Integration Tests
- [x] System Tests

### CODE REVIEW

**Comment by alexstojda** [comment]
I think that's enough changes for 1 PR

File: composer.json [hunk 1]
```diff
@@ -120,6 +120,10 @@
         "ide:full": [
             "@ide",
             "@ide:models"
+        ],
+        "routes":
+        [
+          "@php artisan route:list --columns=name,uri,method"
         ]
```
Comment (EvanDime) [inline composer.json:127]: big fan

**Review by EvanDime — APPROVED** [review]
Glad this got done! Thanks Alex

**Review by Alex-Hayeur — APPROVED** [review]
Looks great, will be super helpful for demonstrating the project.

**Pull request merged**